Source/Core/Core/HLE/HLE.cpp

```diff
@@ -41,7 +42,7 @@ struct SPatch
 };
 
 // clang-format off
-static const SPatch OSPatches[] = {
+constexpr std::array<SPatch, 21> OSPatches{{
```

Comment:
Why not `static` anymore?

Reply:
const (and by extension constexpr) objects already have internal linkage. static is unnecessary here.